packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-authenticator/amplify-authenticator.component.html

```diff
@@ -1,43 +1,3 @@
-<!-- 
-  Define default contents here.
--->
-<ng-template #signIn>
-  <amplify-sign-in></amplify-sign-in>
-</ng-template>
```

Comment:
Default contents no longer need to be defined separately, now that they are explicitly defined where it's being placed.